### PR TITLE
Update profile CTA to match design.

### LINKED ISSUE #2090: Profile empty state has wrong cta
image

Should be like this:
image

### PR DESCRIPTION
**Pull request by kadams54**
Fixes #2090.

### CODE REVIEW

**Comment by kadams54** [comment]
## QA

In order to easily view the "empty profile" page, hack `section-load-watcher.js`, changing line 95 from this:
~~~~
if (this.state.renderEmpty) {
~~~~
To this:
~~~~
if (true) {
~~~~

1. `make run`.
2. Visit `/profile` - you should get the empty page.
3. Verify that the empty page matches the image in #2090.

**Comment by jujugui** [comment]
Test FAILed.
Refer to this link for build results (access rights to CI server needed): 
http://ci.jujugui.org:8080//job/juju-gui/5972/
Test FAILed.

File: jujugui/static/gui/src/app/components/user-profile/empty-user-profile.js [hunk 1]
```diff
@@ -56,21 +56,21 @@ YUI.add('empty-user-profile', function() {
...
-              Your profile is currently empty
-            </h2>
-            <p className="user-profile__empty-text">
-              Your models, bundles, and charms will
-              appear here when you create them.
-            </p>
-          </div>
+            switchModel={this.switchModel}
+            title="Start building"
+            type="positive" />
```
Comment (anthonydillon) [inline jujugui/static/gui/src/app/components/user-profile/empty-user-profile.js]: If you change this to `inline-positive` you can remove the button override.

**Comment by jujugui** [comment]
Test FAILed.
Refer to this link for build results (access rights to CI server needed): 
http://ci.jujugui.org:8080//job/juju-gui/5975/
Test FAILed.

**Comment by anthonydillon** [comment]
Looks good thanks @kadams54

**Comment by jujugui** [comment]
Test FAILed.
Refer to this link for build results (access rights to CI server needed): 
http://ci.jujugui.org:8080//job/juju-gui/5984/
Test FAILed.

**Comment by jujugui** [comment]
Test PASSed.
Refer to this link for build results (access rights to CI server needed): 
http://ci.jujugui.org:8080//job/juju-gui/5986/
Test PASSed.

**Comment by hatched** [comment]
Looks great, thanks @kadams54 

:shipit:

**Comment by jujugui** [comment]
Status: merge request accepted. Url: http://ci.jujugui.org:8080/job/juju-gui-merge

**Pull request merged**